src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -475,9 +545,115 @@ private async Task<ReviewListItemModel> ToggleReviewApproval(ClaimsPrincipal use
             review.ChangeHistory = changeUpdate.ChangeHistory;
             review.IsApproved = changeUpdate.ChangeStatus;
             await _reviewsRepository.UpsertReviewAsync(review);
+            
+            // If this review was approved (first release), check if we should auto-approve namespace for related TypeSpec
+            if (changeUpdate.ChangeStatus && IsSDKLanguage(review.Language))
+            {
+                await CheckAndAutoApproveNamespaceForTypeSpec(review.PackageName, userId);
```

Comment:
This will be an issue. It will mark TypeSpec review as namespace approved when one of the language review is approved.   How does the user request for approval for other languages. We cannot mark other languages as approved when one language review is approved.